less/common.less

```diff
@@ -150,3 +150,20 @@ html.no-js .no-js-warning {
     }
   }
 }
+
+/**********
+* Form
+*/
+
+.optional-text {
+  color: @grey;
+  font-style: italic;
+  font-weight: normal;
+  &:before {
```

Comment:
Doing this so that screenreader will not read out `(` and `)`.   Otherwise it will say something like "bracket, optional, bracket" (if I remember it correctly)